keyboards/dz60/keymaps/olivierko/readme.md

```diff
@@ -1,19 +1,26 @@
 # DZ60
 
 ### _BL:
-Base layer with american ANSI layout.
+Base layer with american ANSI layout.  
```

Comment:
I may not have been clear here, but the sequence "space-space-linebreak" creates the single-line break when rendered to the browser. You don't need the double-line-breaks. (Basically, Lines 5, 10, 15 and 20 aren't needed to get the nice rendering.)

Reply:
Ah, alright. Thanks, hopefully I understood now :)

Reply:
Ah I see it now, used "https://dillinger.io/" and changed once again, was typing blindly.

Reply:
It's a bit tricky – basically every site that implements Markdown has their own way of doing so.

GitHub defines their spec [here](https://github.github.com/gfm/), and the relevant section is [Hard line breaks](https://github.github.com/gfm/#hard-line-breaks).